js/company.js

```diff
@@ -0,0 +1,133 @@
+window.onload = async function startPage() {
+  try {
+    await createMarquee();
+    urlQueryString = new URLSearchParams(window.location.search);
+    companySymbol = urlQueryString.get("symbol");
+    recieveCompanyData(companySymbol);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const recieveCompanyData = async (company) => {
+  toggleLoader();
+  displayPrepToggle();
+  const url = `${Url.profileData}/${company}`;
+  let response = await fetch(url);
+  try {
+    response = await response.json();
+    displayProfile(response.profile);
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+const displayProfile = (data) => {
+  insertProfileHeader(data.image, data.companyName, data.industry);
+  insertStockPrices(data.price, data.changesPercentage);
+  insertDescription(data.description);
+  insertUrl(data.website);
+  getStockHistory();
+  appendChart();
+};
+
+const insertProfileHeader = (imageUrl, name, industry) => {
```

Comment:
consider creating the DOM element on the HTML side and only change the content of the DOM elements.